docs/docsite/rst/playbooks_reuse.rst

```diff
@@ -9,7 +9,7 @@ Creating Reusable Playbooks
 
 While it is possible to write a playbook in one very large file (and you might start out learning playbooks this way), eventually you'll want to reuse files and start to organize things. In Ansible, there are three ways to do this: includes, imports, and roles.
 
-Includes and imports (added in 2.4) allow users to break up large playbooks into smaller files, which can be used across multiple parent playbooks or even multiple times within the same Playbook.
```

Comment:
I think this might still be in the keep "window"

Reply:
Definitely in the keep window.

Reply:
Reinstated